### ISSUE TITLE
 bin/macos/debug/lua  tools/editor/main.lua  cannot open 3rd/ant/tools/fileserver/main.lua: No such file or directory

### ISSUE THREAD

**Issue by darklost** [ejoy/ant#35]
```
connect editor fileserver
(vfs):15: cannot open 3rd/ant/tools/fileserver/main.lua: No such file or directory
stack traceback:
        [C]: in function 'fastio.readall_v'
        (vfs):15: in function 'vfs.read'
        (vfs):21: in function 'loadfile'
        (main.lua):24: in main chunk
        (BOOTSTRAP):11: in main chunk
        [C]: in ?
```

**Comment by aimoonchen** [comment]
We haven't tested editor on macos yet, i will disable fileserver to fix it.

**Comment by aimoonchen** [comment]
@darklost fixed in #[e2b5bf0](https://github.com/ejoy/ant/commit/e2b5bf0bca20145e9075deb30f0a1b04ff017d08)
